internal/compiler/generator/rust.rs

```diff
@@ -1463,6 +1463,15 @@ fn generate_item_tree(
     let item_tree_array_len = item_tree_array.len();
     let item_array_len = item_array.len();
 
+    let element_info_body = if root.has_debug_info {
+        quote!(
+            *_result = self.item_element_infos(_index).unwrap_or_default();
```

Comment:
Can we avoid generating the item_element_info at all if there is no debug info? This should reduce the code size.

Reply:
Sounds good, I’ll make a separate PR.